prow/cmd/deck/jobs.go

```diff
@@ -119,6 +121,24 @@ func (ja *JobAgent) GetJobLog(job, id string) ([]byte, error) {
 	return ja.jc.GetLog(job, num)
 }
 
+func (ja *JobAgent) GetJobLogStream(job, id string, options map[string]string) (io.ReadCloser, error) {
```

Comment:
please document that options is query parameters documented here: https://kubernetes.io/docs/api-reference/v1.7/#read-log